### DISCUSSION TITLE
breakpoint changes

### DISCUSSION THREAD

**Discussion by 563256676** [cefsharp/CefSharp#3877] [General]
When I call the ShowDevTools method, how can I know whether a breakpoint is added or deleted in a JS file, I want to monitor the change of the breakpoint.

**Comment by amaitland** [comment]
`CefSharp` exposes the raw `DevTools Protocol`, you can use the protocol browser online at https://chromedevtools.github.io/devtools-protocol/

Specifically https://chromedevtools.github.io/devtools-protocol/tot/Debugger/ has the list of all events supported and methods related to the debugger. 

I've not tried this myself so no idea if it's even possible. You'll need to do further research yourself.
